srv/components/provisioner/files/glusterfshsaredstorage.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Mount glusterfs sharedstorage
+After=glusterd.service
+Before=salt-minion.service
+Requires=glusterd.service salt-minion.service remote-fs-pre.target local-fs.target
+BindsTo=glusterd.service salt-minion.service
+
+[Service]
+Type=forking
+ExecStart=/usr/libexec/glusterfs/mount-shared-storage.sh
```

Comment:
@SomwanshiAnjali I guess you found some template for that service, are you sure with that command?

Reply:
@andkononykhin2 
for this service I referred details from https://github.com/gluster/glusterfs/commits/master/extras/systemd/glusterfssharedstorage.service.in
Also I have checked this script and it just mounts the glusterfs type mounts which are missing on reboot.

Reply:
@SomwanshiAnjali 
I feel what @andkononykhin2 is trying to say is there's a possibility of this template changing in source which we might miss to capture.
Do evaluate option to insert the lines in existing file, include it if it makes sense; else we continue with template.